### PR TITLE
Constructors / Accessors added

### PR DESCRIPTION
**Pull request by AgoinsUSGS**
Added ControlNetVersioner(FileName) and ControlNetVersioner(QSharedPointer<ControlNet>) constructors.

Added ControlNet::GetLastModified() accessor.

### CODE REVIEW

**Comment by AgoinsUSGS** [comment]
[Create ControlNetVersioner(ControlNet *net) constructor](https://trello.com/c/p8DRUKna/92-create-controlnetversionercontrolnet-net-constructor)

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp [hunk 1]
```diff
@@ -33,14 +33,27 @@ using namespace google::protobuf::io;
 using namespace std;
 
 namespace Isis {
-
+    
   ControlNetVersioner::ControlNetVersioner(QSharedPointer<ControlNet> net) {
-
+    // Populate the internal list of points.
+    for (int i = 0; i < net.GetNumPoints; i++) {
```
Comment (kberryUSGS) [inline isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Should this be a function? Like `net.GetNumPoints()`?

**Comment by AgoinsUSGS** [comment]
Good catch! I have added the () to the method call.

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp (hunk 1 above)
Comment (AgoinsUSGS) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Good catch! I have added the () to the method call.

**Comment by kberryUSGS** [comment]
:+1: Looks good!

**Pull request merged**